install/index.md

```diff
@@ -232,8 +232,8 @@ sections, but there is also a general troubleshooting topic.
    1. Next Topic
 
       After completion of the firmware update with FAS and the preparation of compute nodes, the CSM product stream has
-      been fully installed and configured. Refer to the _HPE Cray EX Installation and Configuration Guide 1.5 S-8000_
-      for other product streams to be installed and configured after CSM.
+      been fully installed and configured. Refer to the _HPE Cray EX System Software Getting Started Guide S-8000_
+      on the HPE Customer Support Center at https://www.hpe.com/support/ex-gsg for more information on other product streams to be installed and configured after CSM.
```

Comment:
I thought we were trying to just say "products" instead of "product streams"?  If so, this comment applies to other areas touched that I won't bother commenting on.

Reply:
Verified with Garima that we are holding off on making this change.